tools/fraken/main.go

```diff
@@ -67,6 +67,7 @@ var (
 	externalVariables = []string{"filepath", "filename", "filetype", "extension", "owner"}
 	maxGoroutines     = 10
 	maxScanFilesize   = 1073741824 /* 1 Gb */
+	minimumScore      = 40
```

Comment:
Do we want to make this a flag and then set that in the `TASK_CONFIG` for this task so that it is configurable via recipes?  I guess it was hard-coded before too though, so maybe that's not needed?

Reply:
Done